python/pyspark/sql/connect/proto/catalog_pb2.pyi

```diff
@@ -49,6 +49,7 @@ else:
 
 DESCRIPTOR: google.protobuf.descriptor.FileDescriptor
 
+@typing_extensions.final
```

Comment:
need to figure out how to remove such annotation

Reply:
What's this annotation for and why we need to remove it ?

Reply:
I don't know what is this annotation, but I think it should not affect other generated files

Reply:
CC @HyukjinKwon Any ideas ?

Reply:
It's autogenerated anyway so I suspect it's fine.

Reply:
It seem this `@typing_extensions.final` is gone, just curious what is the cause?